--- origsrc/ocaml_dbus-0.03/Makefile	2007-09-25 11:05:18.000000000 -0500
+++ src/ocaml_dbus-0.03/Makefile	2008-01-27 12:34:47.437500000 -0600
@@ -34,11 +34,11 @@
 	$(OCAMLC) -a -dllib dlldbus_stubs.so -cclib -ldbus_stubs -cclib -ldbus-1 -o $@ -custom dBus.cmo
 
 dbus_stubs.a: dbus_stubs.o
-	ocamlmklib -o dbus_stubs $+
+	ocamlmklib -o dbus_stubs $+ `pkg-config --libs dbus-1`
 
 libdbus_stubs.a: dbus_stubs.o
 	ar rcs $@ $+
-	ocamlmklib -o dbus_stubs $+
+	ocamlmklib -o dbus_stubs $+ `pkg-config --libs dbus-1`
 
 %.cmo: %.ml
 	$(OCAMLC) -c -o $@ $<
@@ -60,7 +60,7 @@
 
 .PHONY: install
 install: $(LIBS)
-	ocamlfind install -destdir $(OCAMLDESTDIR) -ldconf ignore $(OCAML_PKG_NAME) META $(INTERFACES) $(LIBS) *.a *.so *.cmx
+	ocamlfind install $(OCAML_PKG_NAME) META $(INTERFACES) $(LIBS) *.a *.so *.cmx
 
 uninstall:
 	ocamlfind remove -destdir $(OCAMLDESTDIR) $(OCAML_PKG_NAME)
